src/components/Card/UserGroup/UserGroupCard.jsx

```diff
@@ -0,0 +1,49 @@
+import changeDayFormat from "../../../utils/changeDayFormat";
+import getDate from "../../../utils/getDate";
+import KeywordChip from "../../Chip/KeywordChip";
+import PropTypes from "prop-types";
+
+const UserGroupCard = ({ groupName, keywordList, createdAt, updatedAt }) => {
+  const createdDate = getDate(createdAt);
+  const updatedDate = getDate(updatedAt);
+
+  return (
+    <div className="flex flex-col items-start justify-center gap-3 w-full border-2 border-rose-200/80 bg-white rounded-[30px] px-20 py-10">
+      <p className="flex items-center gap-10">
+        <span className="text-purple-300 text-20">1️⃣ Group Name: </span>
+        <span className="text-rose-400 text-18">{groupName}</span>
+      </p>
+      <p className="flex items-center gap-10">
+        <span className="text-purple-300 text-20">2️⃣ Keyword List: </span>
+        <span className="text-rose-400 text-18">
+          {keywordList.map((keywordName) => {
+            <KeywordChip key={keywordName} keywordName={keywordName} />;
+          })}
+        </span>
+      </p>
+      <p className="flex items-center gap-10">
+        <span className="text-purple-300 text-20">3️⃣ Created date: </span>
+        <span className="text-rose-400 text-18">
+          {`${createdDate.currentYear}년 ${createdDate.currentMonth}월 ${createdDate.currentDate}일 ${changeDayFormat(createdDate.currentDay)} ${createdDate.currentHour}시 ${createdDate.currentMinute}분`}
+        </span>
+      </p>
+      <p className="flex items-center gap-10">
+        <span className="text-purple-300 text-20">4️⃣ Updated date: </span>
+        <span className="text-rose-400 text-18">
+          {updatedDate
+            ? `${updatedDate.currentYear}년 ${updatedDate.currentMonth}월 ${updatedDate.currentDate}일 ${changeDayFormat(updatedDate.currentDay)} ${updatedDate.currentHour}시 ${updatedDate.currentMinute}분`
+            : "수정 내역이 없습니다"}
+        </span>
+      </p>
+    </div>
+  );
+};
+
+export default UserGroupCard;
+
+UserGroupCard.propTypes = {
+  groupName: PropTypes.string.isRequired,
+  keywordList: PropTypes.arrayOf(PropTypes.string).isRequired,
+  createdAt: PropTypes.string.isRequired,
+  updatedAt: PropTypes.any,
```

Comment:
```suggestion
  updatedAt: PropTypes.string,
```
`any`보단 createdAt과 동일하게 `string`으로 설정해주는게 더 좋을 것 같아요!

Reply:
@only-pine @jin-ttao 아 이 부분 얘기해보면 좋겠다고 생각했어요..!
수정되지 않은 경우에는 updatedAt이 어떻게 오나요? 혹시 null로 오지는 않을까 싶어서 확신이 없었는데요. createdAt과 동일하게 오려나요?

Reply:
네, 그룹 생성시에 DB에서 createdAt과 updatedAt 모두 동일한 생성 시간으로 적용됩니다.
그리고 수정시에는 updatedAt만 수정한 시간으로 변경되기 때문에,
`updatedAt: PropTypes.string.isRequired` 로 적용하면 될 것 같습니다!

Reply:
> **결정: 수정 없는 경우 `updatedAt`도 `null`이 아니라 `string`으로 오기 때문에 `createdAt`과 통일하도록 함.**

구두로 이야기 한 부분 남겨둡니다!